document/html/index.bs

```diff
@@ -0,0 +1,389 @@
+        1. [=Upon rejection=] |bodyPromise| with reason |reason|:
+            1. [=Reject=] |returnValue| with |reason|.
+     1. [=Upon rejection=] of |source| with reason |reason|:
+         1. [=Reject=] |returnValue| with |reason|.
+     1. Return |returnValue|.
+</div>
+
+<h2 id="conventions">Developer-facing display conventions</h2>
+
+Browsers, JavaScript engines, and offline tools have common ways of referring to
+JavaScript artifacts and language constructs. For example, locations in
+JavaScript source code are printed in stack traces or error messages, and are
+represented naturally as decimal-format lines and columns in text files. Names
+of functions and variables are taken directly from the sources. Therefore (for
+example) even though the exact format of Error.stack strings does not always
+match, the locations are easily understandable and the same across browsers.
+
+To achive the same goal of a common representations for WebAssembly constructs, the
+following conventions are adopted.
+
+A WebAssembly location is a reference to a particular instruction in the binary, and may be
+displayed by a browser or engine in similar contexts as JavaScript source locations.
+It has the following format:
+
+`${url}:wasm-function[${funcIndex}]:${pcOffset}`
+
+Where
+* `${url}` is the URL associated with the module, if applicable (see notes).
+* `${funcIndex}` is the [=function index=] relative to the module.
+* `${pcOffset}` is the offset in the module binary of the first byte of the instruction, printed in hexadecimal with lower-case digits, with a leading `0x` prefix.
+
+Notes:
+* The URL field may be interpreted differently depending on the
+    context. When the response-based
+    instantiation <a href="#streaming-modules">API</a> is used in a
+    browser, the associated URL should be used; or when the
+    {{ArrayBuffer}}-based instantiation
+    [=instantiate|API=] is used, the browser should represent
+    the location of the API call. This kind of instantiation is analagous to
+    executing JavaScript using `eval`; therefore if the browser has an existing
+    method to represent the location of the `eval` call it can use a similar
+    one for `WebAssembly.instantiate`. For example if the browser uses
+    `foo.js line 10 > eval` or `eval at bar (foo.js:10:3)` for `eval`, it could
+    use `foo.js line 10 > WebAssembly.instantiate` or
+    `WebAssembly.instantiate at bar (foo.js:10:3)`, respectively.
+    Offline tools may use a filename instead.
+* Using hexadecimal for module offsets matches common conventions in native tools
+    such as objdump (where addresses are printed in hex) and makes them visually
+    distinct from JavaScript line numbers. Other numbers are represented in decimal.
+
+While the `name` property of {{ExportedFunction}}
+is specified by the JS API, synthesized function names are also
+displayed in other contexts like devtool callstacks and `Error.stack`.
```

Comment:
`Error.stack` is undefined; there is no such static property. Perhaps you mean `error.stack`.